clientv2/src/components/ProfileViewer/options.ts

```diff
@@ -0,0 +1,311 @@
+export const institutionOptions = [
+  'University of Illinois Urbana-Champaign',
+  'Massachusetts Institute of Technology (MIT)',
+  'Stanford University',
+  'Carnegie Mellon University',
+  'University of California, Berkeley',
+  'California Institute of Technology (Caltech)',
+  'Harvard University',
+  'University of Washington',
+  'Princeton University',
+  'University of California, Los Angeles (UCLA)',
+  'University of Texas at Austin',
+  'Cornell University',
+  'University of Michigan, Ann Arbor',
+  'University of California, San Diego',
+  'University of Wisconsin-Madison',
+  'Georgia Institute of Technology',
+  'University of California, Irvine',
+  'University of Southern California',
+  'University of Maryland, College Park',
+  'Columbia University',
+  'University of Pennsylvania',
+  'University of Chicago',
+  'Yale University',
+  'Duke University',
+  'University of California, Santa Barbara',
+  'University of North Carolina at Chapel Hill',
+  'Purdue University',
+  'University of California, Davis',
+  'New York University (NYU)',
+  'University of Minnesota, Twin Cities',
+  'Rice University',
+  'Northwestern University',
+  'University of Florida',
+  'University of Massachusetts Amherst',
+  'Brown University',
+  'Pennsylvania State University',
+  'University of Virginia',
+  'Johns Hopkins University',
+  'University of Colorado Boulder',
+  'University of Pittsburgh',
+  'Rutgers University',
+  'Ohio State University',
+  'Boston University',
+  'University of Utah',
+  'Washington University in St. Louis',
+  'University of California, Riverside',
+  'Michigan State University',
+  'University of Arizona',
+  'University of Notre Dame',
+  'Indiana University Bloomington',
+];
+
+const csPlusXMajors = [
+  'Computer Science + Animal Sciences',
+  'Computer Science + Crop Sciences',
+  'Computer Science + Education',
+  'Computer Science + Bioengineering',
+  'Computer Science + Physics',
+  'Computer Science + Music',
+  'Computer Science + Anthropology',
+  'Computer Science + Astronomy',
+  'Computer Science + Chemistry',
+  'Computer Science + Economics',
+  'Computer Science + Geography & Geographic Information Science',
+  'Computer Science + Linguistics',
+  'Computer Science + Philosophy',
+  'Computer Science + Advertising',
+  'Mathematics & Computer Science',
+  'Statistics & Computer Science',
+];
+
+export const degreeOptions = [
```

Comment:
Do we just want to break this up into "Bachelors", "Non-thesis Masters", "Researcb Masters", and "PhD"? I don't think people care about BSLAS vs BS vs BFA, and tbh i would concurrently list BS and MCS if I were BS/MCS

Reply:
Also do non-FAANG recruiters even know what MS vs MCS is? (that's why "research" vs "non-research" master's may be the easier term)

Reply:
We can split into bachelors, masters thesis, masters non-thesis, PhD.

Reply:
I think that's the best split for search purposes. Other than places that need ABET accreditation, that's the 4 categories that recruiters will search by.